collect_reprogs.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import filecmp
+import logging
+import os
+import shutil
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Collect C repros programs from subdirs.')
+    parser.add_argument('directory', type=str,
+                        help="Folder to be searched for reprogs.")
```

Comment:
I like the word "reprogs" :) but I think it's a bit unofficial, I would say "C reproducers". That's how we generally call them in the context of syz-manager.

Reply:
done.